Source/SwiftLintFramework/Rules/RuleConfigurations/ExpiringTodoConfiguration.swift

```diff
@@ -0,0 +1,91 @@
+public struct ExpiringTodoConfiguration: RuleConfiguration, Equatable {
+    public struct DelimiterConfiguration: Equatable {
+        public static let `default`: DelimiterConfiguration = .init(opening: "[", closing: "]")
+
+        fileprivate(set) var opening: String
+        fileprivate(set) var closing: String
+    }
+
+    public var consoleDescription: String {
+        return "(approaching_expiry_severity) \(approachingExpirySeverity.consoleDescription), " +
+        "(reached_or_passed_expiry_severity) \(expiredSeverity.consoleDescription)"
+    }
+
+    private(set) var approachingExpirySeverity: SeverityConfiguration
+
+    private(set) var expiredSeverity: SeverityConfiguration
+
+    // swiftlint:disable todo
```

Comment:
nitpick: you can use `disable:next` to avoid line 20

Reply:
thanks—didn't know about this!